### ISSUE TITLE
Fix focus state of close button in modal

### ISSUE THREAD

**Issue by emmawedekind** [Coding-Coach/coding-coach#187]
The focus/hover state of the close button in the modals doesn't have a high enough contrast. I will provide a design of a more contrasted hover/focused state. This is important for keyboard accessibility (the user must be able to see where their cursor/focus is).

![image](https://user-images.githubusercontent.com/7671983/47732427-75724c80-dc66-11e8-817a-230ba067a71f.png)

**Comment by feysalikbal** [comment]
That's true I hardly noticed any change on the focus/hover state of the close button.

**Comment by darrenvong** [comment]
@emmawedekind Does this still need fixing? If so, maybe I can give this a go? Or are you still in the process of designing the hover/focused state for the button?

**Comment by emmawedekind** [comment]
You're welcome to fix it! @darrenvong. Maybe since it's "floating" we can increase the height of the box-shadow to make it look like it's coming more off the page? :)

**Comment by emmawedekind** [comment]
I just realized I never provided a design.. sorry about that! Try my approach in the previous comment and let me know what you think :D

**Comment by darrenvong** [comment]
Just to clarify 😅 Do you mean as the user hover over the button, the height of the background box-shadow should just grow?

**Comment by darrenvong** [comment]
In fact, since the button uses the Button component and I see that we're in the process of migrating to Tailwind (as per issue #232), does that mean this is kinda being blocked by that right now?

**Comment by tureey** [comment]
@darrenvong I'll be pleasured to help you out. I don't have a lot of experience but maybe being together it's easier :+1:

**Comment by darrenvong** [comment]
Sounds good @tureey! I think I'll wait until I know what's going on with the Button component first before trying to work on it.

**Comment by tureey** [comment]
Yes, @darrenvong , He is who is leading all the Frontend development.

**Comment by crysfel** [comment]
@darrenvong #265 has been merged, this issue is no longer blocked. Let me know if you need any help

**Comment by darrenvong** [comment]
@crysfel Awesome, I've been away for the past week so just catching up on things now. I'll take a look at this again soon!

**Comment by emmawedekind** [comment]
Hey @darrenvong any update? :)

**Comment by darrenvong** [comment]
@emmawedekind Just had a chance to look at this! It looks like between your initial bug post and the current version, the close button on the modal no longer floats. Is that intended, or should I turn it back to a floating button again?

Also, when the button is being focused/hovered, its green colour seems to look visibly darker as shown in the following screenshot.

![non-floating-button-hovered](https://user-images.githubusercontent.com/13669362/51571040-dcd01f00-1e98-11e9-9f59-cf2476011344.png)

So here's my follow-up question - is the darker green still not high enough contrast?

**Comment by emmawedekind** [comment]
Are the other buttons in the UI still floating? If so, let's make this one float again. If the others are flat, let's leave it flat. I think a darker green should be okay for a focus/hover state :)

**Comment by darrenvong** [comment]
@emmawedekind As the "Close" button is the only thing on the _modal UI_, the button on it is flat right now. And if you are happy enough with the dark green shown in the screenshot of my previous message, then I guess the bug has fixed itself already! But if the dark green show is not high enough contrast, let me know and it should hopefully be a quick fix :)

**Comment by emmawedekind** [comment]
Is it flat only on focus? As long as we're consistent, I'm okay with it!

**Comment by emmawedekind** [comment]
Ah dang sorry! Closing!
